test/Sentry/SentryLaravelServiceProviderTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Sentry\SentryLaravel;
+
+
+class SentryLaravelServiceProviderTest extends \Orchestra\Testbench\TestCase
+
+{
+
+    protected function getEnvironmentSetUp($app)
+    {
+        // $app['config']->set('services.stripe.secret', getenv('STRIPE_SECRET'));
```

Comment:
left this as an example (from the blog post I pulled this from), though we could remove it